samples/MathSharp.Interactive/MathSharp.Interactive.csproj

```diff
@@ -4,12 +4,6 @@
     <NoWarn>1573;1591;$(NoWarn)</NoWarn>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.0</TargetFramework>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
All of this was duplicated or unnecessary. It is either handled by default or already defined in the root Directory.Build.props